tests/test_multistore_file.py

```diff
@@ -27,6 +34,32 @@ def filename(self):
 
 
 class MultistoreFileTests(unittest.TestCase):
+    def tearDown(self):
```

Comment:
I saw @waprin remove the space between class-def and first method.

Is that a thing? I've always left a space in the presence or absence of a docstring. No need to change, I'm just curious about why do one way or the other.